toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -310,6 +310,7 @@ func (r *RpmRepoCloner) WhatProvides(pkgVer *pkgjson.PackageVer) (packageNames [
 		"provides",
 		provideQuery,
 		fmt.Sprintf("--disablerepo=%s", allRepoIDs),
+		fmt.Sprintf("--releasever=%s", "2.0"),
```

Comment:
Wonder if we can pull in RELEASE_MAJOR_ID instead of hardcoding

Reply:
Yep, that's on my list of todos.

Reply:
nevermind, saw you already had a todo for this